src/impact-dashboard/EpidemicModelContext.tsx

```diff
@@ -52,12 +81,65 @@ function epidemicModelReducer(
 
 function EpidemicModelProvider({ children }: EpidemicModelProviderProps) {
   const [state, dispatch] = React.useReducer(epidemicModelReducer, {
-    stateCode: "US",
-    totalIncarcerated: populationAndHospitalData.US.incarceratedPopulation,
+    stateCode: "US Total",
+    countyName: "Total",
+    countyLevelDataLoading: true,
     rateOfSpreadFactor: 3.7,
     usePopulationSubsets: false,
   });
 
+  // fetch from external datasource
+  React.useEffect(() => {
+    fetch(
+      "https://docs.google.com/spreadsheets/d/e/2PACX-1vSeEO7JySaN21_Cxa7ON_x" +
+        "UHDM-EEOFSMIjOAoLf6YOXBurMRXZYPFi7x_aOe-0awqDcL4KZTK1NhVI/pub?gid=" +
+        "1836987932&single=true&output=csv",
+    ).then(async (response) => {
+      let rawCSV = await response.text();
+      // the first line is not the header row so we need to strip it
+      rawCSV = rawCSV.substring(rawCSV.indexOf("\n") + 1);
+
+      const parsedArray = csvParse(rawCSV, (row) => {
+        const transformedRow = autoType(row);
```

Comment:
I think it might be safer to force-parse cells in each column as a certain type rather than auto-interpret the value type. We don't want a stray character in a spreadsheet throwing a cryptic error much further down the line in the data processing pipeline. Open to thoughts though!

Reply:
I think I actually agree but I am going to punt on this for now .... after this I am going to work on some more updates to the epidemic model, at which point I will know which fields from the spreadsheet we actually care about and will parse them more explicitly